From 7ac03e987ca5dd113e821c923cc510cf9e220ee7 Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Thu, 14 Mar 2013 15:58:46 -0600
Subject: ENGR00254442 Sabre ARD Include WEIM-NOR on imx6_updater_defconfig

Sabre ARD include WEIM-NOR configuration update to
enable higher bandwidth on this interface for mfg tool
support (imx6_updater_defconfig).

Signed-off-by: Alex Sierra <b18039@freescale.com>
---
 arch/arm/configs/imx6_updater_defconfig |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/imx6_updater_defconfig b/arch/arm/configs/imx6_updater_defconfig
index 3de43e6..e562767 100644
--- a/arch/arm/configs/imx6_updater_defconfig
+++ b/arch/arm/configs/imx6_updater_defconfig
@@ -729,6 +729,7 @@ CONFIG_MTD_CFI_UTIL=y
 #
 # CONFIG_MTD_COMPLEX_MAPPINGS is not set
 CONFIG_MTD_PHYSMAP=y
+CONFIG_MTD_IMX6X_WEIMNOR=y
 # CONFIG_MTD_PHYSMAP_COMPAT is not set
 # CONFIG_MTD_ARM_INTEGRATOR is not set
 # CONFIG_MTD_PLATRAM is not set
-- 
1.7.9.5

